jvm/Module.md

```diff
@@ -0,0 +1,4 @@
+# Module bdk-jvm
+The [bitcoindevkit](https://bitcoindevkit.org/) language bindings library for the JVM. Current version: `0.5.1`.
```

Comment:
Same comment as for `android/Module.md`.

```suggestion
The [bitcoindevkit](https://bitcoindevkit.org/) language bindings library for the JVM.
```

Reply:
Done.